### DISCUSSION TITLE
ROCO Kingdom

### DISCUSSION THREAD

**Discussion by 2061360308** [ruffle-rs/ruffle#17007] [Ideas] [answered]
Recently, ROCO Kingdom has been gaining popularity again. When will Ruffle support ROCO Kingdom？
[游戏链接](https://17roco.qq.com)

**Comment by n0samu** [comment]
It looks like the site does not have CORS properly set up to allow Ruffle to load the game. This would need to be fixed by the site owner. https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configure-cors-header

**Reply by 2061360308** [reply]
I disabled CORS in the browser, and SWF resources can now load normally, but there are still problems with subsequent execution.

![扫描件_Errors_001.jpg](https://github.com/ruffle-rs/ruffle/assets/42377282/6340aef9-9fc7-4a22-94e4-a9d11ba6a13f)

![扫描件_Errors_004.jpg](https://github.com/ruffle-rs/ruffle/assets/42377282/3dab7d0a-9737-4297-aaef-fa3656350fcd)

![扫描件_Errors_003.jpg](https://github.com/ruffle-rs/ruffle/assets/42377282/7bd67779-faff-4124-84d9-0fb64c7aaa18)

![扫描件_Loading SWF file_001.jpg](https://github.com/ruffle-rs/ruffle/assets/42377282/afa852dd-e3da-43ff-a21d-793a7890902f)

![扫描件_Errors_002.jpg](https://github.com/ruffle-rs/ruffle/assets/42377282/9023fa39-d4bd-4608-a69e-668a39effc6a)

I am not able to provide a more detailed error report at the moment, but I will upload one in a couple of days if needed.🌝

**Reply by Croworbit** [reply]
Can you make this discussion into an issue? 
Please take all logs and images and put it there! 
😀

**Reply by n0samu** [reply, marked as answer]
Yes this should be an issue, and please provide instructions for disabling CORS to test as well. And it's best to save and attach your console log file rather than uploading screenshots.

**Reply by 2061360308** [reply]
I have created a new issue #17058 . Thank you so much.
